Makefile

```diff
@@ -15,7 +15,7 @@
 
 CFLAGS ?= -O2 -Wall -Wshadow
 
-GIT_VERSION := $(shell git describe --abbrev=6 --dirty --always || date +%F)
+GIT_VERSION := $(shell type git >/dev/null && git describe --abbrev=6 --dirty --always || date +%F)
```

Comment:
I think we can just safely pipe stderr to /dev/null and have it working virtually everywhere.